authenticator/frontend/templates/authenticator/magic_links/landing.html

```diff
@@ -21,11 +21,12 @@ <h1 class="govuk-heading-xl">
     </p>
 
     <div data-qa="claim_magic_link">
-    {{ govukButton({
-      "text":gettext("Continue"),
-      "href": url_for('api_magic_links.use', link_id=link_id, fund=fund_short_name, round=round_short_name),
-      "isStartButton": true })
-    }}
+        {{ govukButton({
+        "text": gettext("Continue") if not has_eligibility else
+        (gettext("Continue") if has_previous_applications else gettext("Start a new application")),
```

Comment:
Maybe a bit easier to read but not too important!
```suggestion
        "text": gettext("Continue") if (not has_eligibility or has_previous_application) else gettext("Start a new application"),
```